Tests/Twig/Fixtures/functions/ez_editorial_yui_config.test

```diff
@@ -7,6 +7,6 @@ ez_platformui_ui_config "twig" function
 --DATA--
 return array('YUIConf' => 'YUI.GlobalConfig')
 --EXPECT--
-YUI.GlobalConfig = {"filter":"min","modules":[]};
-myObj = {"filter":"min","modules":[]};
-{"filter":"min","modules":[]};
+YUI.GlobalConfig = {"filter":"min","modules":[],"root":"","comboBase":"?"};
```

Comment:
To be frank, neither do I :)

I'm not a big fan of just fixing the fixture without understanding what's happening. Can you try to look deeper into it ?